ApiDoctor.Validation/TableSpec/tablespecconverter.cs

```diff
@@ -287,9 +287,10 @@ private TableDecoder FindDecoderFromHeaderText(Stack<Config.DocumentHeader> head
                 else if (kvp.Key.Contains("{x}"))
                 {
                     // eventually bring FriendlyWildcard into this code...
-                    var matchIndex = lastHeader.IndexOf(kvp.Key.Replace("{x} ", string.Empty));
-                    if (matchIndex != -1)
+                    var commonHeaderContent = kvp.Key.Replace("{x} ", string.Empty);
+                    if (lastHeader.EndsWith(commonHeaderContent))
```

Comment:
I think a comment would be nice here to remind you why you made this change.
If I didn't read the reported bug, I wouldn't get it.

Reply:
I've added comment